### PR TITLE
Feature/blog

### PR DESCRIPTION
**Pull request by mraaroncruz**
Please fix the buttons and any other ugliness.
I just spent 3.5 hours on this and am pretty tired of it. Let me know if you need anything else from me.
You can remove redundant or ugly pictures by removing them from the images.yml data file. Moving them to another collection is harder.

### CODE REVIEW

**Comment by mraaroncruz** [comment]
the images that are on their sides in the lightbox are just too big. I will upload some new smaller ones tomorrow.

File: data/images.yml [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+  - IMG_1548
+  - IMG_1550
+  - IMG_1552
+
+wirsing:
+  - DSC02309
+  - DSC02312
+  - DSC02325
+  - DSC02360
+  - IMG_1518
```
Comment (clemenshelm) [inline data/images.yml:101]: Would have probably been less effort to put the images into separate dirs and iterate over them. Makes it also less error-prone when you change something.

**Comment by clemenshelm** [comment]
Wow, that's an ass full of code, as we would say in Austria.

I didn't read every single line, but hey, it's a blog. So had a look at the UI.

What I didn't get at the overview page (at first sight) was that I had to click the headline of the blog to jump into the article. Maybe a link "Read more" would help.

File: source/posts/2015-04-12-post-foodhack-wrapup.md.erb [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+
+## Prizes
+
+Companies gave prizes for the winners that were focused on the winning teams continuing development of their ideas.
+
+### Grand Prize
+
+* 2 days wirsing coaching
+* 40 days working in the co-space coworking space
+* 40 hours working in the Impact Hub coworking space
```
Comment (clemenshelm) [inline source/posts/2015-04-12-post-foodhack-wrapup.md.erb]: I don't think the Co Space and Impact Hub prizes are correct. Where did you get this from?

Comment (clemenshelm) [inline source/posts/2015-04-12-post-foodhack-wrapup.md.erb]: Oh, 5 days times 2 weeks times 4 people. But the winning team were only 3 people.

File: source/posts/2015-04-12-post-foodhack-wrapup.md.erb [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+### 2nd Place Prize
+
+* 20 days working in the coworking space sektor5
+* 4 Organic Product boxes from Biohof Adamah
+* 40 EUR to shop at the Lunzer Massgreisslerei
+
+### 3rd Place Prize
+
+* 12 days at coworking space Stockwerk
+* 4 Zero Waste Jam sirups
```
Comment (clemenshelm) [inline source/posts/2015-04-12-post-foodhack-wrapup.md.erb]: They were also only 3 people.

**Comment by clemenshelm** [comment]
Very extensive and cool! The only thing that bothers me a little is that the outcome is at the very end, which might be the most interesting part of the article. But maybe we can publish little articles in the following weeks where we present the single projects?

File: data/images.yml (hunk 1 above)
Comment (mraaroncruz) [inline data/images.yml:101]: they're on S3

File: source/posts/2015-04-12-post-foodhack-wrapup.md.erb (hunk 2 above)
Comment (mraaroncruz) [inline source/posts/2015-04-12-post-foodhack-wrapup.md.erb]: oh yeah. correct

**Comment by mraaroncruz** [comment]
added read more link and google analytics

**Comment by mraaroncruz** [comment]
I'll upload the smaller images at stockwerk today

**Comment by DominikGuzei** [comment]
There are only images of 3 challenges – if you need all I have the others on my phone!

**Comment by DominikGuzei** [comment]
Very nice work btw! :wink:

**Comment by DominikGuzei** [comment]
I will beautify the buttons now

**Comment by mraaroncruz** [comment]
Helene wrote most of it.

You can give me those photos today. or a dropbox link

**Comment by DominikGuzei** [comment]
I get an error `GET http://localhost:4567/javascripts/lightbox.min.map 404 (Not Found)`, maybe we should also include that or is it only in dev?

**Comment by mraaroncruz** [comment]
ehhh

**Comment by DominikGuzei** [comment]
It takes very long until the page is actually shown because all images are being downloaded before. It would be better to load them after the basic page is shown.

**Comment by mraaroncruz** [comment]
they should be lazy loaded. I will try to figure that out.

**Pull request merged**